advanced/approve_circleci_job/impl.py

```diff
@@ -2,14 +2,29 @@
 from sym.sdk.annotations import hook, reducer
 from sym.sdk.integrations import slack
 from sym.sdk.templates import ApprovalTemplate
+from sym.sdk.notifications import Notification
+from sym.sdk.request_permission import PermissionLevel, RequestPermission
 
 
 # Reducers fill in the blanks that your workflow needs in order to run.
 @reducer
-def get_approvers(event):
-    """Route Sym requests to a specified channel."""
+def get_permissions(event):
+    """
+    approve_deny is set here to allow only the users in the sym-requests slack channel to approve or deny a request
+    allow_self lets the requester approve themself, which is great for testing!
+    """
+    return RequestPermission(
+        webapp_view=PermissionLevel.ADMIN,
+        approve_deny=slack.users_in_channel("#circleci-deploys"),
```

Comment:
yeah that makes sense. I chose to do the coupling to the Slack channel because it was what most closely aligned with the original example.

Reply:
Agree with Ari here. Setting aside the fact that this SDK method doesn't exist yet (but it will), this is really intended as a solution for those are are relying strongly on Slack channels as being the indicator of whether you want someone to be able to approve something, and this is generally not the case; indeed, the primary use case for `slack.users_in_channel()` is for `webapp_view`, not `approve_deny`, as an easy way to represent the scenario of a customer having a private Slack channel where users with approve authority sit and respon to requests, and you want them to easily be able to use the webapp as well, and they aren't already Sym admins.

For the purposes of a broadly-applicable general example like these, I would recommend using `approve_deny=PermissionLevel.MEMBER`.